rustls/src/msgs/handshake.rs

```diff
@@ -1198,16 +1204,7 @@ pub struct ServerHelloPayload {
 
 impl Codec<'_> for ServerHelloPayload {
     fn encode(&self, bytes: &mut Vec<u8>) {
-        self.legacy_version.encode(bytes);
```

Comment:
On the other hand, this is something I might do in a separate commit.

Reply:
I think I'll end up with dead code warnings if I try to do that before implementing confirmation, or I'd have to implement a watered down version that only adds `Encoding::Standard`. In the latter case having that separated seems like it has less value. Do you feel strongly about this one?

Reply:
Nope!